contract_manager/store/contracts/EvmContracts.yaml

```diff
@@ -148,72 +73,141 @@
 - chain: zksync
   address: "0xf087c864AEccFb6A2Bf1Af6A0382B0d0f6c5D834"
   type: EvmContract
-- chain: scroll_sepolia
+- chain: ronin
+  address: "0x2880aB155794e7179c9eE2e38200202908C17B43"
+  type: EvmContract
+- chain: horizen_eon
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: saigon
+- chain: shimmer
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: ronin
-  address: "0x2880aB155794e7179c9eE2e38200202908C17B43"
+- chain: boba
+  address: "0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF"
   type: EvmContract
-- chain: horizen_gobi
+- chain: manta
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: horizen_eon
+- chain: scroll
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: sepolia
-  address: "0x2880aB155794e7179c9eE2e38200202908C17B43"
-  type: EvmContract
-- chain: shimmer
+- chain: chiliz
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: arbitrum_sepolia
+- chain: coredao
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: boba_goerli
+- chain: tomochain
   address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
   type: EvmContract
-- chain: boba
+- chain: base_goerli
+  address: "0xEbe57e8045F2F230872523bbff7374986E45C486"
+  type: EvmContract
+- chain: arbitrum_sepolia
   address: "0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF"
   type: EvmContract
-- chain: manta_testnet
-  address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
+- chain: fuji
+  address: "0x23f0e8FAeE7bbb405E7A7C3d60138FCfd43d7509"
   type: EvmContract
-- chain: manta
-  address: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729"
+- chain: canto_testnet
+  address: "0x26DD80569a8B23768A1d80869Ed7339e07595E85"
   type: EvmContract
-- chain: optimism_sepolia
```

Comment:
just removed this until we deploy again?

Reply:
yeah